content/docs/guides/contribute/contribute-code/frontend-conventions.en.md

```diff
@@ -212,58 +220,67 @@ Dans la terminologie de RTK query, Lorsqu’un composant react appelle un endpoi
 RTK compte le nombre de référence à la même paire (endpoint,{paramètres}). Lorsque deux composants souscrivent à la même donnée. Ils partagent la même clé dans le cache.
 
 ```ts
-import { osrdEditoastApi } from './api.ts'  
-  
-function Component1() {  
-  // component subscribes to the data  
-  const { data } = osrdEditoastApi.useGetXQuery(1)
-  
-  return <div>...</div>  
-}  
-  
-function Component2() {  
-  // component subscribes to the data  
-  const { data } = osrdEditoastApi.useGetXQuery(2)
-  
-  return <div>...</div>  
-}  
-  
-function Component3() {  
-  // component subscribes to the data  
-  const { data } = osrdEditoastApi.useGetXQuery(3)  
-  
-  return <div>...</div>  
-}  
-  
-function Component4() {  
-  // component subscribes to the *same* data as ComponentThree,  
-  // as it has the same query parameters  
-  const { data } = osrdEditoastApi.useGetXQuery(3)  
-  
-  return <div>...</div>  
+import { osrdEditoastApi } from "./api.ts";
+
+function Component1() {
+  // component subscribes to the data
+  const { data } = osrdEditoastApi.useGetXQuery(1);
+
+  return <div>...</div>;
+}
+
+function Component2() {
+  // component subscribes to the data
+  const { data } = osrdEditoastApi.useGetXQuery(2);
+
+  return <div>...</div>;
+}
+
+function Component3() {
+  // component subscribes to the data
+  const { data } = osrdEditoastApi.useGetXQuery(3);
+
+  return <div>...</div>;
+}
+
+function Component4() {
+  // component subscribes to the *same* data as ComponentThree,
+  // as it has the same query parameters
+  const { data } = osrdEditoastApi.useGetXQuery(3);
+
+  return <div>...</div>;
 }
 ```
 
-Ici `Component3` et `Component4` ne vont générer qu’un seul appel vers le back. Ils souscrivent à la même donnée (même endpoint et même paramètre `3`). Ils vont partager la même clé dans le cache. 
+Ici `Component3` et `Component4` ne vont générer qu’un seul appel vers le back. Ils souscrivent à la même donnée (même endpoint et même paramètre `3`). Ils vont partager la même clé dans le cache.
 
 Au total ici il y aura trois appels vers le back, avec les paramètres `1`, `2`, `3`.
 
 Tant qu’il existe au moins un composant react monté, qui appelle le hook `osrdEditoastApi.endpoints.getProjectsByProjectId.useQuery` par exemple, la donnée sera conservée dans le cache.
 
 Dès que le dernier composant est démonté, la donnée est supprimée du cache au bout de 60 secondes (valeur par défaut).
 
+### Translation
+
+Application translation is performed on [Transiflex]. The default language is French.
+If you add a new translation key, it can be added directly to the code, in all available languages. Please note that if you need to correct a translation, we recommend that you use Transiflex, to avoid any conflict.
```

Comment:
It's transifex without the 'l' :)

```suggestion
Application translation is performed on [Transifex]. The default language is French.
If you add a new translation key, it can be added directly to the code, in all available languages. Please note that if you need to correct a translation, we recommend that you use Transifex, to avoid any conflict.
```

Reply:
Done